config/locales/en-PACIFIC.yml

```
@@ -0,0 +1,8 @@
+en:
```

Is this file meant to be added?  I would expect it to come after #97

---

@cjcolvar It's in preparation for #97. I know I need the files, but there might be more work required on the other PR in the short term, so it's setup for when its ready.

---

@prdanelli I added a commit to #97 and did some testing.  Looks like this line should be changed otherwise it will override the default `:en` locale.
```suggestion
en-PACIFIC:
```

---

Thanks @cjcolvar, I've not worked this deep inside the I18n before, so this is useful to know.